qt/scientific_interfaces/Indirect/ConvFitModel.cpp

```diff
@@ -157,9 +157,14 @@ getAnalyser(MatrixWorkspace_sptr workspace) {
         "Could not load instrument resolution from parameter file");
 
   auto component = instrument->getComponentByName(analysers[0]);
-  auto resolutionParameters = component->getNumberParameter("resolution");
-  if (nullptr == component || resolutionParameters.empty())
+  if (component) {
+    auto resolutionParameters = component->getNumberParameter("resolution");
```

Comment:
Could `const` be used here?

Reply:
Yes, actually one should also check if the parameter exists before getting it. Do you mind committing it?